contributing/core/developing.md

```diff
@@ -65,6 +65,18 @@ Or without running the build:
 pnpm pack-next --no-build --release && pnpm unpack-next path/to/project
 ```
 
+Without going through a tarball (only works if you've added the overrides from `pack-next`):
+
+```bash
+pnpm patch-next path/to/project
+```
```

Comment:
But for the overrides from `pack-next`, I need the tarballs, otherwise it points to nonexistent files?

What exactly is the workflow for using `patch-next`?

Reply:
You need to run `pack-next` first and then `pnpm install` with the overrides in place once

After that, you can run `patch-next` without running `pack-next` again